packages/core/utils/package.json

```diff
@@ -25,6 +25,7 @@
     "cross-env": "^5.2.1",
     "express": "^4.18.2",
     "jest": "^29.6.3",
+    "medusa-test-utils": "workspace:^",
```

Comment:
This would need to point to a version, otherwise it'll fail

Reply:
👍

Reply:
Is there any recommended way to install workspace packages? Because `yarn add <package-name>` uses the `workspace` identifier. @sradevski

Reply:
I do a resolution to flie:...

Reply:
I usually just add it manually

Reply:
@thetutlage sometimes I just manually change the version, or when you do `yarn add <package-name>` just specifty @version then.